src/modules/esl-utils/dom/traversing.ts

```diff
@@ -7,14 +8,19 @@ export const isMatches = (el: Element, matcher?: string | ((el: Element) => bool
   return typeof matcher === 'undefined';
 };
 
+/** Safely checks if the target element is within the container element */
+export const isSafeContains = (container: Node | null | undefined, element: Node | null | undefined): boolean => {
+  return isElement(element) && isElement(container) && container.contains(element);
+};
+
 /** Checks that `nodeA` and `nodeB` are from the same tree path */
 export function isRelativeNode(nodeA: null | undefined, nodeB: Node | null | undefined): false;
 /** Checks that `nodeA` and `nodeB` are from the same tree path */
 export function isRelativeNode(nodeA: Node | null | undefined, nodeB: null | undefined): false;
 /** Checks that `nodeA` and `nodeB` are from the same tree path */
 export function isRelativeNode(nodeA: Node | null | undefined, nodeB: Node | null | undefined): boolean;
 export function isRelativeNode(nodeA: Node | null | undefined, nodeB: Node | null | undefined): boolean {
-  return !!(nodeA && nodeB) && (nodeA.contains(nodeB) || nodeB.contains(nodeA));
+  return isSafeContains(nodeA, nodeB) || isSafeContains(nodeB, nodeA);
```

Comment:
It seems the `isElement` check is a little bit too heavy to perform twice on the library level.
```suggestion
  if (!isElement(nodeA) || !isElement(nodeB)) return false;
  return nodeA.contains(nodeB) || nodeB.contains(nodeA);
```